data/sidebars.json

```diff
@@ -280,5 +280,14 @@
         "comparing-systems-cont"
       ]
     }
+  ],
+  "secure-computer-systems": [
+    {
+      "section": "Lecture Notes",
+      "notes": [
+        "getting-started",
+        "design-principles"
```

Comment:
You'll need to make sure that these values match the *filenames* of the corresponding lectures. You'll either need to drop the 01/02 from the filenames in `notes/secure-computer-systems` or add them in here.